how-to/add_k8s_cloud.rst

```diff
@@ -0,0 +1,81 @@
+JAAS: Add a K8S cloud to JIMM
+=============================
+
+Introduction
+------------
+In this how-to, we will demonstrate adding a K8S cloud to JIMM
+and deploying a model.
+
+Prerequisites
+-------------
+
+For this how-to, you will require the following:
+
+- Basic knowledge of Juju
+- Have JIMM deployed with admin access
+- Have a machine based controller added to JIMM
+- Basic knowledge of K8S
+- A controller running in the region next to your K8S deployment, whether that be
+microk8s, EKS, etc.
+
+Ensure kube config is set correctly
+-----------------------------------
+
+Firstly we'll need a local kube config correctly configured
+with the K8S cluster you wish to add to JIMM.
+
+The default location for this file can be found here:
+
+``cat ~/.kube/config``
+
+In this `contexts` object, locate your designed context and remember
+its "name" field.
+
+Updating JIMM to be aware of this cloud
+---------------------------------------
+
+Next we'll add the cloud by context to JIMM.
+
+Firstly let us switch to the JIMM controller (note, this is not the controller JIMM is running on but JIMM itself), run:
+``juju switch <your jimm controller>``
+
+Add your K8S cloud using the following command:
+``juju add-k8s --context-name <context field from ~/.kube/config> --cloud=<cloud name> --region=<region name> <your k8s cloud name>``
```

Comment:
there are a few bits here that you did not explain:
- cloud names the clouds onto which you deployed your k8s cluster
- region names the cloud region into which you deployed your k8s cluster
these two bits of information are used by jimm to determine which controller to add this cloud to in order to make networking as simple as possible

Reply:
I'll put it in but I'm not sure what you mean?